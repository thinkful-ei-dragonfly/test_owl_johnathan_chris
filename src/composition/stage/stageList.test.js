import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import StageList from './stageList.js';

describe('User component', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<StageList stageUsers={[]} />, div);
        ReactDOM.unmountComponentAtNode(div);
      });

    it('renders the UI as expected', () => {
    const tree = renderer
      .create(<StageList stageUsers={[]}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
  });
