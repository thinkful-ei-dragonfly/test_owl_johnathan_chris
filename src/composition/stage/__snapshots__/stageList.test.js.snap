// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User component renders the UI as expected 1`] = `
<div
  className="participants-list"
>
  <ul />
</div>
`;
